libraries/AP_HAL/board/linux.h

```diff
@@ -332,6 +332,19 @@
 #define HAL_GPIO_C_LED_PIN        16
 #define HAL_GPIO_LED_ON           LOW
 #define HAL_GPIO_LED_OFF          HIGH
+#elif CONFIG_HAL_BOARD_SUBTYPE == HAL_BOARD_SUBTYPE_LINUX_EDGE
+#define HAL_BOARD_LOG_DIRECTORY "/var/APM/logs"
+#define HAL_BOARD_TERRAIN_DIRECTORY "/var/APM/terrain"
+#define HAL_INS_DEFAULT HAL_INS_EDGE
+#define HAL_INS_MPU60x0_NAME "mpu60x0"
+#define HAL_INS_MPU60x0_NAME_EXT "mpu60x0ext"
+#define HAL_BARO_DEFAULT HAL_BARO_MS5611_SPI
+#define HAL_BARO_MS5611_NAME "ms5611"
+#define HAL_BARO_MS5611_USE_TIMER false
```

Comment:
this can go away. nobody uses timer anymore

Reply:
Thanks. Totally missed that.